src/app/public/modules/text-editor/defaults/menubar-section-defaults.ts

```diff
@@ -0,0 +1,8 @@
+import {
+  SkyTextEditorMenubarAction
+} from '../types/menubar-action';
+
+export const MENUBAR_ACTION_DEFAULTS: SkyTextEditorMenubarAction[] = [
```

Comment:
Is this internal? If so - we need to add that JS Doc

Reply:
Good catch. Updated.